package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "npm run storybook",
     "test": "eslint src/",
-    "storybook": "start-storybook -p 9001 -c .storybook"
+    "storybook": "start-storybook -p 9001 -c .storybook -s ./src/static"
```

Comment:
Should we really keep this stuff in the source folder? IMHO src is for code only.

Reply:
its actually a good point. Agree with you

Reply:
True, but we need to access files from components, too?

Reply:
I don't understand? Do you mean the storybook files or the components itself? Why don't we just reference the components? AFAIK it is not a requirement to keep storybook and sourcecode both in the src folder to work, but maybe I am misinterpreting.